percona-packaging/scripts/README

```diff
@@ -4,10 +4,10 @@ How to install dependencies:
   sudo ./psmdb_builder.sh --builddir=<absolute path to dir> --install_deps=1
 
 How to get sources from github:
-  bash -x psmdb_builder.sh --builddir=<absolute path to dir> --branch=master --psm_ver=4.0.0 --psm_release=1.1 --mongo_tools_tag=r4.0.0 --get_sources=1
+  bash -x psmdb_builder.sh --builddir=<absolute path to dir> --branch=master --psm_ver=4.2.0 --psm_release=1.1 --mongo_tools_tag=r4.2.0 --get_sources=1
```

Comment:
`--psm_release=1.1` should probably be changed to `--psm_release=1` since we don't have two number Percona version any more